src/styles/index.css

```diff
@@ -6,10 +6,17 @@
 @tailwind utilities;
 
 body {
-  font-family: "Barlow", sans-serif;
-  background: #212121 !important;
+  font-family: var(--font-family-sans), sans-serif;
 }
 
 #metabase-sdk-root {
   background: transparent;
 }
+
+body[data-theme="luminara"] {
+  background-image: url("/light-backdrop.png");
+}
+
+body:not([data-theme="luminara"]) {
+  background: var(--color-background) !important;
```

Comment:
This is because of a bug in the SDK, specifically https://github.com/metabase/metabase/issues/44802. We delete this line once the bug has been fixed. TL;DR: EmbedFrame's style leaked.